config/7.12.0/pim/process-migration.yaml

```diff
@@ -88,20 +95,104 @@ processMigration:
     - metadata:
         name: "[[.ApplicationName]]-process-migration"
       data:
-        project-overrides.yml: |-
+        application.yaml: |-
+          # Default pim configuration
+          quarkus:
```

Comment:
No need to re-define the same properties that are already part of the application, only those that are to be overridden or added.

Reply:
I've removed already locally.

Reply:
I choose this way to keep things easier to user and added some comments as well to the configuration to instruct user what to change, if needed.

Reply:
That might bring to some confusion for non experienced users IMO but I don't have anything against it.

Reply:
@jakubschwan what do you think about it?
I am ok with both as well, but having the default config here as well might be easier to user to customize the config if needed.

Reply:
Here should be default configuration, right? And then in override parts will depend on configuration.
As we have used this pattern - default configuration and specific config overrides it when needed - for all other cases. I would suggest to keep to this pattern. As Ruben mentioned, this whole configuration in all cases can bring some confusion.
Comments are nice thing to have, and I think we can keep them :)

Reply:
ok, but, I think we can't override the config-map content, each, external, mysql and postgresql have its own content.